tests/models/wav2vec2/test_tokenization_wav2vec2.py

```diff
@@ -145,8 +146,10 @@ def test_tokenizer_decode_added_tokens(self):
             [24, 22, 5, tokenizer.word_delimiter_token_id, 24, 22, 5, 77, tokenizer.pad_token_id, 34, 34],
         ]
         batch_tokens = tokenizer.batch_decode(sample_ids)
+        batch_tokens_2 = tokenizer.batch_decode(sample_ids, skip_special_tokens=True)
 
         self.assertEqual(batch_tokens, ["HELLO<unk>!?!?$$$", "BYE BYE<unk>$$$"])
+        self.assertEqual(batch_tokens_2, ["HELO!?!?", "BYE BYE"])
```

Comment:
let's add a new token, like `"<new_tokens>"` and test that we can encode decode as we expect for both fast and slow tokenizers! 🤗

Reply:
Is there a fast `Wav2Vec2CTCTokenizer`? I can't not find it!

Reply:
Sorry there is no fast tokenizer here! my bad. Let's just use other tokens than `<unk>`!

Reply:
Sorry for the late. I updated it.